pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1622,6 +1626,17 @@ func validateDisks(field *k8sfield.Path, disks []v1.Disk) []metav1.StatusCause {
 				Field:   field.Index(idx).Child("cache").String(),
 			})
 		}
+
+		// Verify disk and volume name can be a valid container name since disk
+		// name can become a container name which will fail to schedule if invalid
+		e := validation.IsDNS1123Subdomain(disk.Name)
+		if len(e) != 0 {
+			causes = append(causes, metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueInvalid,
+				Message: fmt.Sprintf("Invalid value: \"%s\": %s, regex used for validation is '%s'", disk.Name, dns1123LabelErrMsg, dns1123LabelFmt),
```

Comment:
Can't you just reuse the error message of the validation for the Message field? That would eliminate the need for copying `dns1123LabelErrMsg` and `dns1123LabelFmt` constants. By the way, you copied dns11223LABEL consts, but use dns1123SUBDOMAIN validation, they are slightly different.

Reply:
Oops. Intellisense enter error, meant to use the dns1123 func, will change though.